doc-examples/hibernate-example-kotlin/build.gradle

```diff
@@ -28,4 +28,6 @@ dependencies {
     runtimeOnly mnSql.micronaut.jdbc.tomcat
     runtimeOnly mnLogging.logback.classic
     runtimeOnly mnSql.h2
+
```

Comment:
@dstepanov Just a note about this. Had to add this, hibernate >= 6.3.x with Kotlin (if com.fasterxml.jackson is on the classpath) is loading jackson KotlinModule which requires kotlin-reflect to be on the classpath. This should not be an issue as micronaut-starter by default adds kotlin-reflect to the classpath.